static/app/views/organizationStats/usageTable.tsx

```diff
@@ -104,17 +122,27 @@ class UsageTable extends Component<Props> {
         {formatUsageWithUnits(dropped, dataCategory, getFormatUsageOptions(dataCategory))}
       </CellStat>,
       <CellStat key={5}>
-        <Button
-          title="Go to project level stats"
-          data-test-id={project.slug}
-          size="xs"
-          onClick={() => {
-            this.loadProject(parseInt(stat.project.id, 10));
-          }}
-        >
-          <StyledIconGraph type="bar" size="sm" />
-          <span>View Stats</span>
-        </Button>
+        {isProjectSelected(
+          stat.project,
+          this.props.selectedProjects,
+          ProjectsStore.getAll()
+        ) ? (
+          <Button disabled size="xs">
+            {t('Project is selected')}
+          </Button>
+        ) : (
+          <Button
+            title="Go to project level stats"
+            data-test-id={project.slug}
+            size="xs"
+            onClick={() => {
+              this.loadProject(parseInt(stat.project.id, 10));
+            }}
+          >
+            <StyledIconGraph type="bar" size="sm" />
+            <span>{t('View Stats')}</span>
```

Comment:
the `View Stats` button takes you to the single project view in stats. wouldn't disabling the button in multi-project mode prevent users from selecting a single project from the table?

Reply:
Ah yes, good point. I wonder if we want to maybe show that the project is selected in a different way 🤔